projects/storefrontapp/src/app/spartacus/spartacus-features.module.ts

```diff
@@ -399,6 +399,7 @@ if (environment.cpq) {
         allPageMetaResolversEnabledInCsr: true,
         sciEnabled: true,
         useExtendedMediaComponentConfiguration: true,
+        showRealTimeStockInPDP: false,
```

Comment:
[MINOR]
It would be good to enable all new breaking changes in our storefrontapp (not in defaultFeatureToggles!). See [Coding Guidelines](https://wiki.one.int.sap/wiki/display/spar/Coding+Guidelines#CodingGuidelines-AddaFeatureToggleinthecodesourcecode).

That said, I guess you cannot enable it now because the newest backend version (including the new productAvailabilities endpoint) is not deployed on the Spartacus CI server. And I guess it's also the reason why the e2e suite is marked as "fleaky".

If it's the case, please create a ticket for:
- enabling the toggle in the storefrontapp spartacus-features.module (not in defaultFeatureToggles!)
- removing the "flaky" marker from e2e, so the e2e suite is included in the CI pipeline
... when a newest backend is deployed on CI backend instance.

Btw. which backend version do you need?

Reply:
Noted wil create the tickets.
The backend API for ProductAvailabilities will be released as a part of FP7 (GA -> December).

Reply:
Tickets : [CXSPA-8948](https://jira.tools.sap/browse/CXSPA-8948)
[CXSPA-8947](https://jira.tools.sap/browse/CXSPA-8947)